source/walkthroughs/wt_active_learning.rst

```diff
@@ -0,0 +1,227 @@
+Step-by-Step Guide to Active Learning with Log Data in MindMeld
+===============================================================
+
+Active Learning is an approach to continuously and iteratively pick data that is most informative for the models to train on from a pool of unlabelled data-points. By picking such data-points, active learning provides a higher chance of improving the accuracy of the model with fewer training examples. It also greatly reduces the number of queries to be annotated by humans. MindMeld provides this inbuilt functionality to select these must-have queries from existing data or additional logs and datasets to get the best out of your conversational applications.
```

Comment:
The American spelling is "labeled", also we should remove the hyphens in "data point": "pool of _unlabeled data points_" and "By picking such _data points_"